CONTRIBUTING.md

```diff
@@ -15,57 +16,89 @@ generated with markdown-toc</a></i></small>
-     |                                                    |                                                                                  |
-     |                                                    |      Return Conjur access token encrypted by Conjur server certificate, or error |
-     |                                                    |<---------------------------------------------------------------------------------|
-     |                                                    |                                                                                  |
-     |     Return Conjur access token encrypted over mTLS |                                                                                  |
-     |<---------------------------------------------------|                                                                                  |
-     |                                                    |                                                                                  |
-     | Request secret values using token                  |                                                                                  |
-     |-------------------------------------------------------------------------------------------------------------------------------------->|
-     |                                                    |                                                                                  |
-     |                                                    |                                                   Return secret values, or error |
-     |<--------------------------------------------------------------------------------------------------------------------------------------|
-     |                                                    |                                                                                  |
-     | Return secret values, or error                     |                                                                                  |
-     |--------------------------------------------------->|                                                                                  |
-     |                                                    |                                                                                  |
-```
+### Sequence diagrams
+
+It may help to understand some typical flows when this Puppet module is
+included in your Puppet workflows. The following Conjur Puppet module
+workflows are portrayed below:
+
+- [Using Conjur Host Identity with Host Factory](#using-conjur-host-identity-with-host-factory)
+- [Using Windows Registry / Windows Credential Manager Pre-Provisioning](#using-windows-registry--windows-credential-manager-pre-provisioning)
+- [Using Host Identity with API Key Configured in Puppet Manifest](#using-host-identity-with-api-key-configured-in-puppet-manifest)
+
+#### Using Conjur Host Identity with Host Factory
```

Comment:
I think this first (and possibly third) diagram lacks important information about the flow after the first convergence. I believe that the initial identity establishment is correct but after that first HFT-based identity establishment, a second run of the agent will use the facts from the host to establish identity and the server only fetches the secret.

Reply:
Self-fixed